how-to/customize-workspace/client/src/modules/init-options/interop/init-options.ts

```diff
@@ -35,12 +36,16 @@ async function shareContext(payload: ShareContextPayload) {
 	}
 }
 
-export async function init(options: unknown, loggerCreator: LoggerCreator) {
-	logger = loggerCreator("InitOptionsInteropHandler");
+export async function initialize(definition: ModuleDefinition, createLogger: LoggerCreator, helpers?: never) {
+	logger = createLogger("InitOptionsInteropHandler");
 	// the init function could be passed limits (e.g. only support the following intents or contexts. Only publish to the following context groups etc.)
 	logger.info("The handler has been loaded");
 }
 
+export function supportedActions(): string[] {
```

Comment:
Why is this hardcoded? If we wanted to register more handlers we would have to change the code? (github is showing a collapsed view so I may be missing something).

Reply:
Moved to options